doc/development_process/release_process.rst

```diff
@@ -232,70 +232,94 @@ created.
 Tagging
 =======
 
-.. note::
-
-    This section uses tagging 1.11.0-rc1 as an example, replace with the
-    appropriate version.
-
-Every time a release candidate (or the final release) needs to be tagged, the
-following steps need to be followed:
+The final release and each release candidate shall be tagged using the following
+steps:
 
 .. note::
 
     Tagging needs to be done via explicit git commands and not via GitHub's release
     interface.  The GitHub release interface does not generate annotated tags (it
     generates 'lightweight' tags regardless of release or pre-release).
 
-#. Update the :zephyr_file:`VERSION` file in the root of the Git repository. If it's a
-   release candidate, use ``EXTRAVERSION`` variable::
+.. tabs::
+
+    .. tab:: Release Candidate
+
+        .. note::
+
+            This section uses tagging 1.11.0-rc1 as an example, replace with
+            the appropriate release candidate version.
+
+        #. Update the version variables in the :zephyr_file:`VERSION` file
+           located in the root of the Git repository to match the version for
+           this release candidate. The ``EXTRAVERSION`` variable is used to
+           identify the rc[RC Number] value for this candidate::
 
-    EXTRAVERSION = rc1
+            EXTRAVERSION = rc1
 
-#. Commit the update to the :zephyr_file:`VERSION` file, use ``release:`` as a commit
-   tag.
-#. Check that CI has completed successfully before tagging.
-#. Tag and push the version, using annotated tags:
+        #. Commit and post a PR of the updated :zephyr_file:`VERSION` file using
+           ``release: Zephyr 1.11.0-rc1`` as the commit subject. Verify that CI
+           has completed successfully.
```

Comment:
Shouldn't the PR also be merged before proceeding to the tagging step?

Reply:
yes